oak_containers_orchestrator/src/container_runtime.rs

```diff
@@ -94,6 +97,8 @@ pub async fn run(container_bundle: &[u8]) -> Result<(), anyhow::Error> {
         serde_json::from_str(&oci_filesystem_bundle_config_file)?
     };
 
+    log::info!("Setting up container");
```

Comment:
maybe, if easy and helpful. Is there a way we can identify the container here, e.g., print the container name or so?

e.g. `log::info!("Setting up container from {}", CONTAINER_DIR)`.

Reply:
We don't have identifying information like a name in the Runtime bundle. The root directory is also named the same for all container images, so I don't think that provides useful information.